Webclient/src/app/module/viewer/module/raid_graph/component/raid_graph/raid_graph.scss

```diff
@@ -91,7 +91,7 @@
     background: $tdc_darker;
     border-radius: 3px 0 0 3px;
     padding: 0.5 * $spacing 0.5 * $spacing;
-
+    padding-left: 56px;
```

Comment:
Does that work for all device sizes?

Reply:
Hmm after inspection the padding needed depends on the Y axis of the graph. So if the Y axis has 1000 or 1000000, it will have to be padded accordingly ....

Lets close this PR and I'll open new one if I find a solution

Reply:
ok